lib/timeline/component/item/Item.js

```diff
@@ -294,6 +294,74 @@ class Item {
     }
   }
 
+	    /**
+	     * Repaint a control button on the top right of the item when the item is selected
+	     * @param {HTMLElement} anchor
+	     * @protected
+	     */
+
+    }, {
+	    key: "_repaintControlButton",
```

Comment:
Something is wrong here. I think you mistakenly copy-pasted from a compiled code.
This should be written like the above functions. Please revise.

Reply:
Yes you´re right. I have corrected the errors. Thankyou